django_fido/static/django_fido/js/fido2.js

```diff
@@ -41,6 +41,9 @@ function base64_to_bytes(base64) {
     for (var i = 0; i < len; i++) {
         bytes[i] = binary_string.charCodeAt(i);
     }
+    if (typeof activeFidoWindow === 'function') {
+        activeFidoWindow();
+    }
```

Comment:
Hook is good enough, but it shouldn't be placed into a conversion function. I'd rather put it into `fido2*RequestCallback`s below.

Reply:
Hook is moved to fido2*RequestCallback